SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/GridBasedLBStrategy/GridBasedLBStrategyTest.cpp

```diff
@@ -348,3 +348,12 @@ GRIDBASEDLBSTRATEGY_TEST(GIVEN_two_cells_WHEN_actor_in_one_cell_THEN_strategy_re
 
 	return true;
 }
+
+GRIDBASEDLBSTRATEGY_TEST(zz_CleanUp_GridBasedLBStrategyTest)
```

Comment:
haha, using `zz` to make sure it's run last?

do we need the deployment up and running for these tests?

Reply:
zz to be doubly sure it's last 😄 
I checked with these this and Owenership...Test and they both require the deployment